Samples/BasicVariant.tests.json

```diff
@@ -6,7 +6,9 @@
             "Result": "false"
         },
         "Variant": {
-            "Result": "default"
+            "Result": {
+                "ConfigurationValue": "default"
```

Comment:
@mrm9084 We should verify both variant name and configuration value.

Reply:
@rossgrambo, can you make this change, so we don't have a huge merge conflict with each other.